test/reorderError.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit Main Test Suite</title>
+	<link rel="stylesheet" href="../dist/qunit.css">
+	<script src="../dist/qunit.js"></script>
+	<script>
+	// When failed test that checked test call count.
+	// Ref #907
+	// This issues has 2 case.
+	// First Case.
+	// 1. First test failed and second test successed in first module.
+	// 2. First test failed in second module.
+	// 3. Test count should be 3.
+	
+	// Given
+	var totalCount1 = 0;
+	sessionStorage["qunit-test-First fail module-should be fail"] = 1;
```

Comment:
This will fail on (old) environments that lack `sessionStorage`. Maybe they can skip the whole test.